tapir/coop/views/membership_resignation.py

```diff
@@ -52,13 +52,15 @@ class Meta:
         ]
         sequence = [
             "share_owner",
-            "resignation_type",
             "cancellation_reason",
             "cancellation_date",
             "pay_out_day",
             "paid_out",
             "add_buttons",
         ]
+        exclude = [
+            "resignation_type"
+        ]
```

Comment:
I thought this would be useful to show in the table, do you think it's too much?

Reply:
Ya, 'cause it's already indicated at the in the paid_out column. Don't you think it's not enough?

Reply:
Ah yeah I missed that

Reply:
New commit done